docs/user/unsupported/mbstyle.rst

```diff
@@ -19,14 +24,544 @@ Quick example of module usage (API change):
 Internally the extension provides greater access to the parsed style:
 
 .. code-block:: java
-  
+
     MBStyle mbstyle = MapBoxStyleParser.parse( reader );
 
     // pull back all layers for a provided source
     List<MBLayer> layers = mbstyle.layers( "mapbox://mapbox.mapbox-streets-v6" );
 
     // pull back selected layers for a provided source
-    List<MBLayer> layers = mbstyle.layers( "mapbox://mapbox.mapbox-streets-v6", "water" ); 
+    List<MBLayer> layers = mbstyle.layers( "mapbox://mapbox.mapbox-streets-v6", "water" );
 
     //Which can be used to Generate Styler Layer Descriptor:
-    StyleLayerDescriptor sld = MBStyleTransformer.transform( layers );
\ No newline at end of file
+    StyleLayerDescriptor sld = MBStyleTransformer.transform( layers );
+
+MapBox Types
+^^^^^^^^^^^^
+ copied from the `MapBox Style Specification <https://www.mapbox.com/mapbox-gl-js/style-spec/>`_
```

Comment:
What does this look like when compiled? I expect it should followed by a newline for rst formatting to work properly.

Also, should start with a capital letter (?)